base_tier_validation_formula/models/tier_definition.py

```diff
@@ -12,7 +12,7 @@ class TierDefinition(models.Model):
         help="Write Python code that defines when this tier confirmation "
              "will be needed. The result of executing the expresion must be "
              "a boolean.",
-        default="""# Available locals:\n#  - rec: current record""",
+        default="""# Available locals:\n#  - rec: current record\nrec""",
```

Comment:
If it must be a boolean, why not True?

Reply:
@etobella Thank you for suggestion. I changed this code.